squirrel_maze/resources/menus.py

```diff
@@ -93,9 +92,10 @@ def combat_menu():
     else:
         if answers['selection'] == 'goblin':
             actors = []
-            actors.append(sm_actor.Actor(actor_id=len(actors), name='ham', pc_type='pc', level=1, max_hp=10, max_str=10,
-                          max_dex=10, max_sta=10))
-            actors.append(sm_npc.get_big_goblin(actor_id=len(actors)))
+            db = sm_db_helpers.Database('squirrel_maze/data/db.json')
```

Comment:
I don't really like opening the database here. Should probably do this when the game launches and make it a global thing.

Reply:
Also, I need to close the database somewhere.